### PR TITLE
Added missing TransportMode field to accompany existing TransportSubmode

### PR DESCRIPTION
**Pull request by seime**
TransportMode and TransportSubmode usually come in pairs several places in NeTEx. 
In parameter assignments/validityParameters it is missing, instead there is a VehicleMode.

This PR adds TransportMode to accompany TransportSubmode

The PR has been discussed with @nick-knowles that will verify that it aligns well with other parts of Netex.

### CODE REVIEW

**Comment by skinkie** [comment]
This requires a documentation update.

image

**Review by skinkie — APPROVED** [review]

File: xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd [hunk 1]
```diff
@@ -379,6 +382,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Vehicel Modes to which ACCESS RIGHTs  apply.</xsd:documentation>
```
Comment (ue71603) [inline xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd:382]: typo: Vehicle

File: xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd [hunk 2]
```diff
@@ -379,6 +382,11 @@ Rail transport, Roads and Road transport
...
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="TransportMode" type="AllVehicleModesOfTransportEnumeration" minOccurs="0">
```
Comment (ue71603) [inline xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd:385]: I am a bit unsure about this one: What is the difference between the TransportMode and the previous VehicleModes?
And what would the single TransportMode/TransportSubmode pair express in the context of the OrganisationValidityParametersGroup?

File: xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd (hunk 1 above)
Comment (skinkie) [reply xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd:382]: Already seems to be resolved.

**Comment by skinkie** [comment]
@ue71603 can you approve this one, typo is also fixed in my merge.

**Review by ue71603 — APPROVED** [review]

**Comment by ue71603** [comment]
@skinkie  as we decided not to do #330  should we do this one? But then we are stuck with not answering https://github.com/NeTEx-CEN/NeTEx/issues/300

**Comment by skinkie** [comment]
What is the reason VehicleModes is prefered of TransportModes @Aurige ?

**Comment by Aurige** [comment]
you need to select one or the other, I just favour the original Transmodel over the TPEG compatibility one (own point of view here)

File: xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd (hunk 2 above)
Comment (ue71603) [reply xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd:385]: @Aurige  I can't add a comment that this is for TPEG compatibility. Either you add it or we do it and afterwards think about it.

**Review by ue71603 — APPROVED** [review]

**Comment by seime** [comment]
Since PR was originally created, a new element `TransportModes` has arrived; 

```
        <xsd:element name="TransportModes" type="AllModesListOfEnumerations" minOccurs="0">
          <xsd:annotation>
            <xsd:documentation>ANy mode to which Mode validitry parameters apply + v1.2.2</xsd:documentation>
          </xsd:annotation>
        </xsd:element>
```

This is of another enum again

**Comment by nick-knowles** [comment]
Added to UML 
image

**Comment by ue71603** [comment]
main part was already there. closed here and example and typo done here: https://github.com/NeTEx-CEN/NeTEx/pull/618

**Comment by trurlurl** [comment]
Removing "needs documentation update" label since closed without having been merged.

**Pull request closed without merging**